doc/rados/configuration/ceph-conf.rst

```diff
@@ -323,48 +342,49 @@ For example:
     [global]
     line = "to be, or not to be"
 
-In option values, there are four characters that are treated as escape
-characters: ``=``, ``#``, ``;`` and ``[``. They are permitted to occur in an
-option value only if they are immediately preceded by the backslash character
-(``\``). For example:
+There are four metacharacters that must be escaped with a backslash (``\``)
+if they are meant to be part of the option's value: ``=``, ``#``, ``;`` and ``[``.
+
+Example:
 
 .. code-block:: ini
 
     [global]
-    secret = "i love \# and \["
+    secret = "I l0ve \# and \["
 
-Each configuration option falls under one of the following types:
+Each configuration option specifies one of the following types for its value:
 
 .. describe:: int
 
-   64-bit signed integer. Some SI suffixes are supported, such as "K", "M",
-   "G", "T", "P", and "E" (meaning, respectively, 10\ :sup:`3`, 10\ :sup:`6`,
-   10\ :sup:`9`, etc.). "B" is the only supported unit string. Thus "1K", "1M",
-   "128B" and "-1" are all valid option values. When a negative value is
-   assigned to a threshold option, this can indicate that the option is
-   "unlimited" -- that is, that there is no threshold or limit in effect.
+   A 64-bit signed integer. Some SI suffixes are supported, including ``K``, ``M``,
+   ``G``, ``T``, ``P``, and ``E``.  These represent, respectively, 10\ :sup:`3`, 10\ :sup:`6`,
+   10\ :sup:`9`, etc.). ``B`` (bytes)is the only supported unit string. Thus ``1K``, ``1M``,
+   ``128B`` and ``-1`` are all valid option values. When a negative value is
+   assigned to an option that defines a threshold or limit, this often indicates that the value is
+   "unlimited" -- that is, no threshold or limit will be enforced. Options that
+   allow such a value will usually indicate so in their individual description text.
```

Comment:
s/will indicate so/will indicate as much/

But even this, I think, is too conversational and relies too much upon a deep familiarity with the quirks of spoken English.

So I tentatively suggest this:

"Options that allow unlimited values are described as allowing unlimited values in their respective description texts."